### ISSUE TITLE
Double Click Pull Request leads to error

### ISSUE THREAD

**Issue by geeForceOne** [github/VisualStudio#2314]
## Versions
 - GitHub Extension for Visual Studio version:  2.8.1.7307
 - Visual Studio version:  16.0.0 (and 15.9.11)

## What happened
When trying to double click a pull request, i get the following error (see screenshot).
We are using github enterprise.

### Steps to Reproduce
Team Explorer  -> Pull Requests -> Double Click a PR
Hitting the Refresh button afterwards leads to Object Reference not set.. (SC 3)

### Screenshots
![image](https://user-images.githubusercontent.com/1780268/55479458-6da44300-561e-11e9-9869-96d87c7d96e9.png)
![image](https://user-images.githubusercontent.com/1780268/55479469-71d06080-561e-11e9-9d5a-06f28d67cf70.png)
![image](https://user-images.githubusercontent.com/1780268/55479490-7ac13200-561e-11e9-93f8-3846228220c0.png)

### Logs
see email

**Comment by geeForceOne** [comment]
@meaghanlewis let me know if you need more information.

**Comment by meaghanlewis** [comment]
I don't think we do at this time, thanks @geeForceOne. Unfortunately we are a small team and it will likely take us some time to investigate this issue in more detail.

**Comment by geeForceOne** [comment]
@meaghanlewis FYI we migrated from github enterpirse 2.11.X to 2.15.9. now it works. For me the issue is solved. If you want to keep the issue open for further reference feel free, if not it can be closed :-)
